evm/v0.5/testv2/GetterSetter.test.ts

```diff
@@ -0,0 +1,105 @@
+import { assertBigNum } from '../src/matchers'
+import * as h from '../src/helpers'
+import { GetterSetterFactory } from '../src/generated/GetterSetterFactory'
+import { makeTestProvider } from '../src/provider'
+import { assert } from 'chai'
+import { Instance } from '../src/contract'
+import { ethers } from 'ethers'
+
+const getterSetterFactory = new GetterSetterFactory()
+
+const provider = makeTestProvider()
+let roles: h.Roles
+
+beforeAll(async () => {
+  const rolesAndPersonas = await h.initializeRolesAndPersonas(provider)
+
+  roles = rolesAndPersonas.roles
+})
+
+describe('GetterSetter', () => {
+  const requestId =
+    '0x3bd198932d9cc01e2950ffc518fd38a303812200000000000000000000000000'
+  const bytes32 = ethers.utils.formatBytes32String('Hi Mom!')
+  const uint256 = ethers.utils.bigNumberify(645746535432)
+
+  let gs: Instance<GetterSetterFactory>
+  const deployment = h.useSnapshot(provider, async () => {
+    gs = await getterSetterFactory.connect(roles.defaultAccount).deploy()
+  })
+
+  beforeEach(async () => {
+    await deployment()
+  })
+
+  describe('#setBytes32Val', () => {
+    it('updates the bytes32 value', async () => {
+      await gs.setBytes32(bytes32)
+
+      const currentBytes32 = await gs.getBytes32()
+      assert.deepEqual(h.toUtf8(currentBytes32), h.toUtf8(bytes32))
+    })
+
+    it('logs an event', async () => {
+      const tx = await gs.connect(roles.stranger).setBytes32(bytes32)
+      const receipt = await tx.wait()
+      const args: any = receipt.events?.[0].args
```

Comment:
Isn't `args: any` kind of redundant?

Reply:
@j16r It's actually not, because from the generated typings, `.args` is `any[]`

Reply:
Isn't `any` the "world" type, i.e. `any[]` is a member of `any`? i.e. what happens if you don't put `any` here?

Reply:
Line 51 will complain with `Property 'from' does not exist on type 'any[]'.`